modules/ROOT/examples/swift/code-snippets/SampleCodeTest.swift

```diff
@@ -706,6 +707,45 @@ class SampleCodeTest {
         replicator.start()
         // end::getting-started[]
     }
+    
+    func dontTestPredictionQuery() throws {
```

Comment:
As per discussions

- [ ] Use a 'fake' classifier model as example with an image input and a dictionary of category/probability as output.   Sentiment analysis model does not align with the inputs in example
- [ ] Start with description of PredictiveModel interface and sample implementation of predict function 
- [ ] Describe each API in separate function (a) model registration (b) index creation (c) prediction function and query (d) reregistration.